packages/compass-web/package.json

```diff
@@ -56,7 +56,8 @@
     "bson": "^6.2.0",
     "mongodb": "^6.3.0",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "@mongodb-js/compass-connections": "^1.25.1"
```

Comment:
This is a dev dep, not peer

Reply:
We use both ConnectionsManager and ConnectionsManagerProvider so it should be a production dep but our rule asks us to have the prod deps same as peer deps hence I did this.

Reply:
The rule youre describing only applies to plugin packages, it should not be validating imports in this package, there is no `compass-scripts check-peer-deps` in the package.json commands. Do you see it failing if you remove those packages from here? This is unexpected

Reply:
Ok this makes sense now. I should've also checked what's behind the `check` script, thank you for explaining. It does not fail, as expected.

Reply:
Addressed this in my recent commit.